### ISSUE TITLE
[Assistive tech and device support] Page functionality isn't keyboard accessible. (11.05.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#29338]
#### VFS team name
Education Application (BAH)

#### VFS product name
Comparison Tool Redesign

#### Point of Contact/Reviewers
Trevor Pierce (Accessibility)

---

### Platform Issue
Page functionality isn't keyboard accessible.

### Issue Details
Search by Name and Search by Location tabs are not keyboard focusable. Can't switch from one to the other without clicking using a mouse or tapping.

### Link, screenshot or steps to recreate
![Screen Shot 2021-08-26 at 5 02 33 PM](https://user-images.githubusercontent.com/934879/131042062-841b7fd1-4fa7-45b6-8807-9753118c1280.png)

### VA.gov Experience Standard
[Cateogy Number 11, Issue Number 05](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.1.1_A

---

### Platform Recommendation
Add links and proper tab markup to these so they are keyboard focusable and clickable.

https://www.w3.org/TR/wai-aria-practices-1.2/examples/tabs/tabs-1/tabs.html

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by AngelaFowler82** [comment]
As a screen reader user, I was unable to access the "Search by Name" and "Search by Location" tabs using either the Tab key or the Form Field navigation key. I was able to arrow to it and click it, but only because I already knew it was there. The screen reader does not indicate that these are clickable elements. In my opinion, this should be a defect level 1, as a screen reader user who does not already know these elements exist will likely miss them.

**Comment by dmccurdybah** [comment]
@AngelaFowler82 were you testing on the branch in the linked PR or on staging?

**Comment by joshkimux** [comment]
@dmccurdybah Angela doesn't have access to the branch in the linked PR-- is there any chance we could get her a staging link for a second review?

**Comment by dmccurdybah** [comment]
@joshkimux This is still in our internal review process, so when it passes our QA approval then it'll be merged and deployed to staging.

**Comment by joshkimux** [comment]
@dmccurdybah thank you for clarifying! Would you be able to message Angela when it's deployed to staging for a second accessibility review? Angela is a screen reader user, so notifications on when things have been deployed can get easily lost/missed. Direct messaging would be preferred by an @ or over slack if possible 🙇

**Comment by dmccurdybah** [comment]
@joshkimux I can tag Angela here when this is ready for her review 👍

**Comment by delli-gatti-michael** [comment]
@AngelaFowler82 The fix for this is on staging and ready for review here:
https://staging.va.gov/gi-bill-comparison-tool-sandbox/

**Comment by AngelaFowler82** [comment]
Confirming that this is fixed. I can now navigate between "Search by name" and "Search by location" using the keyboard.
